default_translations/de/messages.de.yml

```diff
@@ -212,3 +201,37 @@ Bad credentials.: Fehlerhafte Zugangsdaten
 
 custom: gewählten
 random: zufälligen
+
+mail:
```

Comment:
When you touch translations anyway, please use the opportunity to break lines at a reasonable length.